azkaban-exec-server/src/main/java/azkaban/execapp/FlowPreparer.java

```diff
@@ -158,6 +158,11 @@ void setup(final ExecutableFlow flow) throws ExecutorManagerException {
       FileIOUtils.deleteDirectorySilently(tempDir);
       LOGGER.error("Error in preparing flow execution {}", flow.getExecutionId(), ex);
       throw new ExecutorManagerException(ex);
+    } catch (final Throwable t) {
+      FileIOUtils.deleteDirectorySilently(tempDir);
+      LOGGER.error("Error in preparing flow execution {}", flow.getExecutionId(), t);
```

Comment:
Could you check the interface of this and above `error` methods? We might not be getting the stack trace of the exception but rather the `toString()` result of the object.

Reply:
Thank you for reviewing. It is giving full cause of the issue.

Reply:
Thanks for checking! I got confused while reading some logs locally.